clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/DisableServiceAtomicOperation.java

```diff
@@ -48,26 +45,53 @@ private void disableService() {
     String account = description.getAccount();
     String cluster = getCluster(service, account);
 
-    updateTaskStatus(
+    DescribeScalableTargetsRequest describeRequest =
+        new DescribeScalableTargetsRequest()
+            .withServiceNamespace(ServiceNamespace.Ecs)
+            .withResourceIds(String.format("service/%s/%s", cluster, service))
+            .withScalableDimension(ScalableDimension.EcsServiceDesiredCount);
+    DescribeScalableTargetsResult describeResult =
+        autoScalingClient.describeScalableTargets(describeRequest);
+
+    if (isSuspended(describeResult)) {
+      updateTaskStatus(
+          String.format(
+              "Autoscaling already suspended for server group %s for %s.", service, account));
+    } else {
+      updateTaskStatus(
         String.format("Suspending autoscaling on %s server group for %s.", service, account));
-    RegisterScalableTargetRequest suspendRequest =
+      RegisterScalableTargetRequest suspendRequest =
         new RegisterScalableTargetRequest()
-            .withServiceNamespace(ServiceNamespace.Ecs)
-            .withScalableDimension(ScalableDimension.EcsServiceDesiredCount)
-            .withResourceId(String.format("service/%s/%s", cluster, service))
-            .withSuspendedState(
-                new SuspendedState()
-                    .withDynamicScalingInSuspended(true)
-                    .withDynamicScalingOutSuspended(true)
-                    .withScheduledScalingSuspended(true));
-    autoScalingClient.registerScalableTarget(suspendRequest);
-    updateTaskStatus(
+          .withServiceNamespace(ServiceNamespace.Ecs)
+          .withScalableDimension(ScalableDimension.EcsServiceDesiredCount)
+          .withResourceId(String.format("service/%s/%s", cluster, service))
+          .withSuspendedState(
+            new SuspendedState()
+              .withDynamicScalingInSuspended(true)
+              .withDynamicScalingOutSuspended(true)
+              .withScheduledScalingSuspended(true));
+      autoScalingClient.registerScalableTarget(suspendRequest);
+      updateTaskStatus(
         String.format("Autoscaling on server group %s suspended for %s.", service, account));
+    }
 
     updateTaskStatus(String.format("Disabling %s server group for %s.", service, account));
     UpdateServiceRequest request =
         new UpdateServiceRequest().withCluster(cluster).withService(service).withDesiredCount(0);
     ecs.updateService(request);
     updateTaskStatus(String.format("Server group %s disabled for %s.", service, account));
   }
+
+  private boolean isSuspended(DescribeScalableTargetsResult describeResult) {
+    if (describeResult.getScalableTargets() != null
```

Comment:
Might want to check if describeResult is null (if the api call fails for some reason, it's possible that describeResult is null)